cpu/stm32/stmclk/Makefile

```diff
@@ -1,17 +1,22 @@
 MODULE = stm32_clk
 
-SRC = stmclk_common.c
 
-ifneq (,$(filter $(CPU_FAM),f2 f4 f7))
-  SRC += stmclk_f2f4f7.c
-else ifneq (,$(filter $(CPU_FAM),f0 f1 f3))
-  SRC += stmclk_f0f1f3.c
-else ifneq (,$(filter $(CPU_FAM),l0 l1))
-  SRC += stmclk_l0l1.c
-else ifneq (,$(filter $(CPU_FAM),l4 wb))
-  SRC += stmclk_l4wb.c
-else ifneq (,$(filter $(CPU_FAM),g0 g4))
-  SRC += stmclk_gx.c
+ifneq (,$(filter $(CPU_FAM),mp1))
+  SRC = stmclk_mp1.c
+else
+  SRC = stmclk_common.c
```

Comment:
I would do this differently:

```mk
ifneq (mp1,$(CPU_FAM))
  SRC = stmclk_common.c
endif

ifneq (,$(filter $(CPU_FAM),f2 f4 f7))
  SRC += stmclk_f2f4f7.c
[...]
else ifneq (,$(filter $(CPU_FAM),mp1))
  SRC += stmclk_mp1.c
endif
```

If that works, that would be a smaller change.